operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/GenericKubernetesResourceMatcher.java

```diff
@@ -192,15 +192,15 @@ public static <R extends HasMetadata, P extends HasMetadata> Result<R> match(R d
     for (int i = 0; i < wholeDiffJsonPatch.size() && matched; i++) {
       var node = wholeDiffJsonPatch.get(i);
       if (nodeIsChildOf(node, List.of(SPEC))) {
-        matched = match(specEquality, node, ignoreList);
+        matched = match(valuesEquality, node, ignoreList);
       } else if (nodeIsChildOf(node, List.of(METADATA))) {
         // conditionally consider labels and annotations
         if (considerMetadata
             && nodeIsChildOf(node, List.of(METADATA_LABELS, METADATA_ANNOTATIONS))) {
           matched = match(labelsAndAnnotationsEquality, node, Collections.emptyList());
         }
       } else if (!nodeIsChildOf(node, IGNORED_FIELDS)) {
-        matched = match(true, node, ignoreList);
+        matched = match(valuesEquality, node, ignoreList);
```

Comment:
So that's the only "real" change, right?

Reply:
yes